Dropdown/src/Dropdown.test.js

```diff
@@ -0,0 +1,136 @@
+jest.mock("./style.less", () => jest.fn());
+jest.useFakeTimers();
+
+import React, {PropTypes} from "react";
+import Dropdown from "./Dropdown";
+import Collapse from "react-collapse";
+
+import {configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15.4';
+import toJson from 'enzyme-to-json';
+
+import {shallow, mount, render} from "enzyme";
+
+configure({adapter: new Adapter()});
+
+const options = [
+    {value: "first", label: "First Label", searchableValue: "first"},
+    {value: "second", label: "Second Label", searchableValue: "second"},
+    {value: "third", label: "Third Label", searchableValue: "third"}
+];
+
+describe("Dnn Dropdown component", () => {
+
+    it("Renders with minimal setup", () => {
+
+        const wrapper = shallow(<Dropdown options={[]} value={null} onSelect={f => f}/>);
+        expect(wrapper).toMatchSnapshot();
+
+    });
+
+    it("Renders labels properly according to props", () => {

```

Comment:
When looking these tests it's hard to tell what is behaviour/appearance actually being tested. If you have multiple `expect` calls at different places it could indicate that such tests should be split into multiple cases with a more focused purpose.

Also, I'd encourage you to follow a pattern for adding comments to separate the three stages of each unit test:

```
// Arrange
// set up your subject under test here

// Act
// perform the tested action

// Assert
// assertions pg here
```

When this becomes a habit it's easier to write concise and precise tests.

Another pattern, which I personally prefer is to write `given`/`when`/`then` but for consistency with existing back-end tests let's stick to the above structure.

Reply:
ok! 👍